01_install_requirements.sh

```diff
@@ -19,7 +19,7 @@ if [ -z "${METAL3_DEV_ENV}" ]; then
   # TODO -- come up with a plan for continuously updating this
   # Note we only do this in the case where METAL3_DEV_ENV is
   # unset, to enable developer testing of local checkouts
-  git reset ee1b2fec44e22700c8d250d5e0e371e3fd0aba17 --hard
+  #git reset 67afcb3315b44e2c68121493a819819966e588a2 --hard
```

Comment:
This needs to be removed  - note you can use `METAL3_DEV_ENV` to reference a local source checkout for testing with latest metal3-dev-env  :)